wavefront_sdk/common/utils.py

```diff
@@ -101,6 +101,35 @@ def is_blank(string):
     # return len(re.sub(r'[\s]+', '', s)) == 0
 
 
+def validate_tags(tags):
+    """
+    Ensure that the tag is not empty, otherwise throw the error.
+
+    :param tags:
+    :return:
+    """
+    for tag in tags:
+        if is_blank(tag):
+            raise ValueError('Event tag: ' + tag + ' cannot be blank')
+
+
+def validate_annotations(annotations):
+    """
+    Ensure that annotation key and value are not empty.
+
+    Otherwise throw the error.
+
+    :param annotations:
+    :return:
+    """
+    for key, value in annotations.items():
+        if is_blank(key):
+            raise ValueError('Annotation key: ' + key + ' cannot be blank')
```

Comment:
Same logging an empty key does not help. Just say `Annotation key cannot be blank`